### PR TITLE
integrate messaging backend with frontend. Fixed

### LINKED ISSUE #180: integrate messaging backend with frontend.
integrate messaging backend with frontend.

### PR DESCRIPTION
**Pull request by Hemu21**
## Description

Messaging platform is integrated with backend.

## Issue

[Issue Link](https://github.com/Kajol-Kumari/InfluMart/issues/180) resolve #180

### CODE REVIEW

**Comment by Hemu21** [comment]
@RohitGupta1237 @Kajol-Kumari can you please check.

File: backend/controllers/connectController.js [hunk 1]
```diff
@@ -24,33 +26,50 @@ const allRequests = async (req, res) => {
...
-  await Request.findByIdAndDelete(requestId);
-  // Emit hello message to both users
-  req.io.to(request.sender._id.toString()).emit('receiveMessage', helloMessage);
-  req.io.to(request.receiver._id.toString()).emit('receiveMessage', helloMessage);
-  res.status(200).json({ message: "Request accepted and hello message sent" });
+
+  await Promise.all([
+    autoMessage.save(),
+    conversation.save(),
+    Request.findByIdAndDelete(requestId),
```
Comment (Kajol-Kumari) [inline backend/controllers/connectController.js:69]: why this?

File: backend/controllers/influencerController.js [hunk 2]
```diff
@@ -99,7 +99,7 @@ exports.signup = async (req, res) => {
 // Login as an influencer
 exports.login = async (req, res) => {
   const { username, password } = req.body;
-
+  console.log(username,password)
```
Comment (Kajol-Kumari) [inline backend/controllers/influencerController.js:102]: remove it please

File: backend/middleware/brandorInfluencerAuthenticationMiddleware.js [hunk 3]
```diff
@@ -7,18 +7,18 @@ const InfluencerSignupRequest = require("../model/influencerSignupRequestModel")
...
 
   // Extract token from header
   const token = authHeader.split(' ')[1];
-
+  console.log(token)
   try {
     // Verify token
     const decoded = await promisify(jwt.verify)(token, process.env.JWT_SECRET_KEY);
-
+    console.log(decoded)
```
Comment (Kajol-Kumari) [inline backend/middleware/brandorInfluencerAuthenticationMiddleware.js:21]: these are debugging consoles, can u please remove?

**Review by Kajol-Kumari — CHANGES_REQUESTED** [review]
as it has backend changes as well,can u please share screen recording of the functionality?

**Pull request merged**